src/ontology/cl-edit.owl

```diff
@@ -31137,6 +31161,106 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
+
+# Class: obo:CL_4033028 (diffuse bipolar 2 cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25309346") Annotation(oboInOwl:hasDbXref "PMID:26500507") Annotation(oboInOwl:hasDbXref "PMID:27833534") obo:IAO_0000115 obo:CL_4033028 "An OFF diffuse bipolar cell that predominantly connects to ON parasol cells and lateral amacrine cells. This cell contains a large number of synaptic ribbons and a small axon arbor area.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033028 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033028 "2023-04-04T10:49:07Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033028 "DB2 cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") rdfs:comment obo:CL_4033028 "In primates, a diffuse bipolar 2 cell makes an average of 133 ribbon synapses, 47 contacts are with OFF parasol cells and 4 contacts are with OFF midget cells.")
+AnnotationAssertion(rdfs:label obo:CL_4033028 "diffuse bipolar 2 cell")
+SubClassOf(obo:CL_4033028 obo:CL_0000750)
+
+# Class: obo:CL_4033029 (diffuse bipolar 3a cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") Annotation(oboInOwl:hasDbXref "PMID:33009001") obo:IAO_0000115 obo:CL_4033029 "An OFF calbindin-positive bipolar cell that has a large dendritic field and stratifies narrowly close to the middle of the inner plexiform layer. Its axon terminal is characterized by regularly branching and varicose processes resembling beads on a string. Most of DB3a contacts with cones are triad-associated.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033029 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033029 "2023-04-04T10:53:33Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.3389/fnana.2015.00122") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033029 "DB3 cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033029 "DB3a cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.3389/fnana.2015.00122") Annotation(oboInOwl:hasDbXref "PMID:32032773") rdfs:comment obo:CL_4033029 "Originally, DB3a cells were termed diffuse bipolar 3 (DB3) cells.When an additional bipolar type which stratifies slightly more vitread and is calbindin negative was detected, this cell was renamed diffuse bipolar 3a cell. The axons of neighbouring DB3a cells make homologous gap junctions. The DB3a cells provide the majority of their output to OFF parasol cells.")
```

Comment:
- Add space before "When ..."
- neighbouring --> neighboring to follow American English spelling standard of CL